cmd/up/activate.go

```diff
@@ -547,6 +555,7 @@ func (up *upContext) waitUntilAppIsAwaken(ctx context.Context, app apps.App) err
 
 			// If the app is not sleeping anymore, we are done
 			if _, ok := appToCheck.ObjectMeta().Annotations[model.StateBeforeSleepingAnnontation]; !ok {
+				up.analyticsMeta.WaitedForAwakenDuration(time.Since(startWaitingForAwaken))
```

Comment:
This isn't super exact since sometimes the deployment might awaken by the time we hit this part of the code, but I think it's good enough to measure the user experience (did they have to wait for the deployment to wake up?).  Appreciate advise if there's a better place to put this.